### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by ttahm3d**
- Added documentation for avatar in avatar/index.html
- changed the assets path for image tags

Merge after pull request #8

### CODE REVIEW

**Review by coldpigli — COMMENTED** [review]
Looks Good.

File: docs/avatar/index.html [hunk 1]
```diff
@@ -76,7 +76,155 @@ <h2 class="heading-3 head-lvl-3" id="available-components">
...
+								alt="avatar 5"
+								loading="lazy"
+								class="avatar-round avatar-xl"
+							/>
+						</div>
+					</div>
+				</div>
+				<div class="code-example">
+					<pre>
+						<code class="html">
```
Comment (kjais1720) [inline docs/avatar/index.html:134]: You can use JS to replace all the '<' with '&lt' in the code example, it's difficult to read the code rn

Comment (ttahm3d) [reply docs/avatar/index.html]: Thanks for helping Krituraj. I have changed the code for it.

**Comment by ttahm3d** [comment]
- changed codeblock formatting as per @kjais1720 comment.

File: docs/alert/index.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+		/>
+		<link
+			rel="stylesheet"
+			href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/night-owl.min.css"
+			integrity="sha512-i5X6Fdn/ZqvGSqPrdMa3FgcpXM/Nr6YccSKFYT93zljl/HZDEpvBbE5Pxp91eiWGccZLrL/LDQJd7fjTRYsVaA=="
+			crossorigin="anonymous"
+			referrerpolicy="no-referrer"
+		/>
+		<link rel="stylesheet" href="../../css/style.css" />
+		<title>Chunks</title>
```
Comment (junaidshaikh-js) [inline docs/alert/index.html:20]: You can rename the title of every page related to the component like Chunks | Alert 

Something like this, just a suggestion.

**Review by junaidshaikh-js — COMMENTED** [review]
Everything else looks good.

File: docs/alert/index.html (hunk 2 above)
Comment (ttahm3d) [reply docs/alert/index.html:20]: Thanks for bring this up. I totally forgot about it while refactoring

**Pull request merged**